### PR TITLE
Verify that Greek translations are working

### PR DESCRIPTION
**Pull request by amandabeiner**
#### Description:
Why:
As a product intended for users in Greece and/or Cyprus, we want to make
sure that the Greek translations are working on the exposure history
page, and that our translations are up to date with Lokalise.

This commit:
Ran the i18n update method to make sure we had the latest translations,
and confirmed that the Greek translations are appearing on the
ExposureHistory flow.

Note that we are still missing some translations (they are Empty in
Lokalise) for this part of the app. But we have verified that the
ExposureHistory screen is correctly set up to receive Greek translations
since the headline has been correctly translated.

#### Linked issues:

Addresses: [Validate Greek translations](https://trello.com/c/I6D1GdoR/279-validate-that-greek-translations-are-able-to-be-added)

#### Screenshots:

Simulator Screen Shot - iPhone 11 - 2020-08-06 at 16 12 39

#### How to test:
- set HA to `GR`
- Change system language to Greek
- Validate that the headline at the Exposure History page is correctly translated to Greek

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -138,7 +138,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "org.pathcheck.covidsafepaths.bt"
+        applicationId "org.pathcheck.gr.bt"
```
Comment (juancruzgs) [inline android/app/build.gradle:141]: This change shouldn't be committed, we use the general app ID `org.pathcheck.covidsafepaths.bt` for debugging and the specific app IDs are changed during the CI process

File: android/app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
+  <!-- Channel label for notifications shown in App > Notifications > Path Check BT -->
+  <string name="notification_channel_description">إشعارات PathCheck BT</string>
```
Comment (juancruzgs) [inline android/app/src/main/res/values-ar/strings.xml:4]: We no longer need this string, it's not being used

File: ios/COVIDSafePaths.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1198,7 +1198,7 @@
 				);
 				MARKETING_VERSION = 0.0.1;
 				OTHER_SWIFT_FLAGS = "$(inherited) -D COCOAPODS -Xcc -fmodule-map-file=\"${PODS_CONFIGURATION_BUILD_DIR}/Alamofire/Alamofire.modulemap\" -Xcc -fmodule-map-file=\"${PODS_CONFIGURATION_BUILD_DIR}/RealmSwift/RealmSwift.modulemap\" -Xcc -fmodule-map-file=\"${PODS_CONFIGURATION_BUILD_DIR}/ZIPFoundation/ZIPFoundation.modulemap\" -Xcc -fmodule-map-file=\"${PODS_ROOT}/Headers/Private/Realm/Realm.modulemap\" -D DEBUG";
-				PRODUCT_BUNDLE_IDENTIFIER = org.pathcheck.bt;
```
Comment (johnschoeman) [inline ios/COVIDSafePaths.xcodeproj/project.pbxproj:1201]: we probably dont want to commit the changes to this file.

**Review by johnschoeman — APPROVED** [review]
LGTM

File: android/app/src/main/res/values-ar/strings.xml (hunk 2 above)
Comment (aledustet) [reply android/app/src/main/res/values-ar/strings.xml]: @amandabeiner let me know when you're back at this and we can go on a lokalise journey to clean up some stuffffff

**Comment by amandabeiner** [comment]
Going to close this since these pulls are happening automatically on CI, and we have verified that they are working on iOS and Android with #155

**Pull request closed without merging**